scripts/deploy/deployStakedTokenWrapper.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers } from 'hardhat';
+import { DeployUpgradeFunc } from '../../types/types';
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  const { tribalChief } = addresses;
+
+  const BENEFICIARY = process.env.BENEFICIARY;
+
+  if (!BENEFICIARY) {
+    throw new Error('BENEFICIARY environment variable contract address is not set');
+  }
+
+  if (!tribalChief) {
+    throw new Error('TribalChief environment variable contract address is not set');
```

Comment:
`tribalchief` isn't an env.

Reply:
This is how the scripts I modeled this one from behaved.

Reply:
fixed